integration/create_test.go

```diff
@@ -72,15 +69,14 @@ func TestCreateStress(t *testing.T) {
 			}
 
 			// Attempt to create a collection for Tigris with a schema.
-			// If we get an error, that's MongoDB (FerretDB ignores that argument for non-Tigris handlers),
-			// so we create collection without schema.
+			// If we get an error that support for "validator" is not implemented, that's Postgres.
+			// If we get an error that "$tigrisSchemaString" is unknown, that's MongoDB.
+			// In both cases, we create a collection without a schema.
 			err := db.CreateCollection(ctx, collName, &opts)
 			if err != nil {
-				var cmdErr *mongo.CommandError
-				if errors.As(err, &cmdErr) {
-					if strings.Contains(cmdErr.Message, `unknown top level operator: $tigrisSchemaString`) {
-						err = db.CreateCollection(ctx, collName)
-					}
+				if strings.Contains(err.Error(), `support for field "validator" is not implemented yet`) ||
```

Comment:
I think we should create a helper method like `errorTextContains(error, string)` and use it everywhere in tests where needed instead of `strings.Contains`. That would help us understand where we don’t return proper errors yet.